docs/advanced/settings.md

```diff
@@ -424,6 +424,13 @@ Specify an IP range in the IPv4 CIDR format. The number of IPs must be four time
 }
 ```
 
+**Supported options and values**:
+
+The value is a JSON object string containing these members:
+- `repository`: The repository name of the support bundle image.
+- `tag`:  The tag name of the support bundle image.
+- `imagePullPolicy`: Support bundle image pull policy. The value can be in `IfNotPresent`, `Always`, and `Never`. Please see the [container document](https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy) in Kubernetes for more information.
```

Comment:
```suggestion
The value is a JSON object literal that contains the following key-value pairs:

- `repository`: Name of the repository that stores the support bundle image.
- `tag`:  Tag assigned to the support bundle image.
- `imagePullPolicy`: Pull policy of the support bundle image. The supported values are `IfNotPresent`, `Always`, and `Never`. For more information, see [Image pull policy](https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy) in the Kubernetes documentation.
```

Reply:
I followed the JSON terminology guidelines outlined [here](https://www.w3schools.com/JS/js_json_objects.asp).

Reply:
Thanks, apply the change and squashed all commits.